### ISSUE TITLE
Intellisense - duplicates

### ISSUE THREAD

**Issue by pzentner** [Microsoft/AL#585]
After a while, I am getting the same suggestions multiple times in the list. For instance, I am adding an action to a new page and in the beginning I am seeing a list like this (correct):

*AccessByPermissions
*ApplicationArea
*CaptionML
*Description

Now, I can't yet find a pattern, but after a while, I am seeing (in the same area - and after making several changes):

*AccessByPermissions
*AccessByPermissions
*AccessByPermissions
*AccessByPermissions
*ApplicationArea
*ApplicationArea
*ApplicationArea
*ApplicationArea
*CaptionML
*CaptionML
*CaptionML
*CaptionML
*Description
*Description
*Description
*Description

So, basically, everything is shown 4 times. When I restart VSCode, it's fine again. This is happening with CodeLens = true. I am trying now to turn it off and monitor, if this has anything to do with it.

**Comment by pzentner** [comment]
Without codelens it just started happening as well. But now, I also got the following errors in the output window together with the message "The AL server crashed 5 times in the last 3 minutes...".

Process is terminated due to StackOverflowException.
[Info  - 4:32:55 PM] Connection to server got closed. Server will restart.

Process is terminated due to StackOverflowException.
[Info  - 4:33:05 PM] Connection to server got closed. Server will restart.

Process is terminated due to StackOverflowException.
[Info  - 4:33:14 PM] Connection to server got closed. Server will restart.

Process is terminated due to StackOverflowException.
[Info  - 4:33:26 PM] Connection to server got closed. Server will restart.

Process is terminated due to StackOverflowException.
[Error - 4:33:38 PM] Connection to server got closed. Server will not be restarted.

When I restart VSCode again, it works somewhat, but now the rest is going haywire - probably due to the AL Server not running anymore (trying to restart it now). Getting things like this:

```
severity: 'Error'
message: 'The name 'ddrraazziiWWeetteellppmmooCC' does not exist in the current context'
at: '58,28'
source: 'AL'
```

or 

```
severity: 'Error'
message: 'The name 'ddrraazziiWWeetteellppmmooCC' does not exist in the current context'
at: '58,28'
source: 'AL'
```

and I don't think this is just some foreign language.

But one other thing I was noticing: When restarting VSCode, it all worked fine (Intellisense) until I created a new action manually (not using taction). As soon as the first warning showed up (I am using CodeAnalysis), I started getting duplicated suggestions.

**Comment by dkatson** [comment]
The same issue in https://github.com/Microsoft/AL/issues/540
I found this in wizard pages. Also after adding action or group.

**Comment by DominikDitoIvosevic** [comment]
Could it be that you have duplicate extensions installed? This usually happens to me when I mess something up and have 2 versions installed

**Comment by pzentner** [comment]
No, I only have 1 version of an extension installed. However, I haven't seen this anymore for the past week or so, after I upgraded to the CTP14 version of AL. I am going to keep monitoring, if this still happens. What does still happen is that it gets a little confused in Wizard or other pages with a lot of groups and then it slows down and takes a while to figure out what actual errors are or not (so error highlighting doesn't catch up).

**Comment by epernst** [comment]
I can confirm that this "The AL server crashed 5 times in the last 3 minutes..." happens with Wizard pages and the October build.

**Comment by gudrunleos** [comment]
same experience here as well. - Might this be related to the _al.enablecodeanalysis = true_ ?

**Comment by atoader** [comment]
Has anybody experienced this in the latest release?

**Comment by gudrunleos** [comment]
Not me :)

**Comment by epernst** [comment]
I have not experienced this error again since the October build.

**Comment by atoader** [comment]
Thanks for replying! I will close this.
